content/post/110-go-ipfs-0-7-0.md

```diff
@@ -0,0 +1,80 @@
+
+### Rotating keys
+
+Along with switching the default key type, we've added support for rotating Identity keys. If you would like to change the key type of your IPFS node, you can now do so with the rotate command. **NOTE: This will affect your Peer Id, so be sure you want to do this!** Your existing identity key will be backed up in the Keystore so that it can still be referenced for things like IPNS records.
+
+```bash
+ipfs key rotate -o my-old-key -t ed25519
+```
+
+## 📦 Key export/import
+
+Speaking of backing up keys, we've added commands to allow you to export and import keys from the IPFS Keystore to a local .key file. This does not currently apply to the IPFS identity key, `self`, which is housed in the configuration file.
+
+```bash
+ipfs key gen mykey
+ipfs key export -o mykey.key mykey # ./<name>.key is the default path
+ipfs key import mykey mykey.key # on another node
+```
+
+## #️⃣ IPNS paths now encode the key name as a base36 CIDv1 by default
+
+Previously go-ipfs encoded the key names for IPNS paths as base58btc multihashes (e.g. Qmabc...). We now encode them as base36 encoded CIDv1s as defined in the [peerID spec](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#string-representation) (e.g. k51xyz...) which also deals with the encoding of public keys. This is nice because it means that IPNS keys will by default be case-insensitive and that they will fit into DNS labels (e.g. k51xyz...ipns.localhost) and therefore that subdomain gateway redirections (e.g. from localhost:8080/ipns/{key} to {key}.ipns.localhost) will look better to users in the default case.
+
+Many commands will accept a `--ipns-base` option that allows changing command outputs to use a particular encoding (i.e.  base58btc multihash, or CIDv1 encoded in any supported base)
+
+## 📮  Multiaddresses now accept PeerIDs encoded as CIDv1
+
+In preparation for eventually changing the default PeerID representation multiaddresses can now contain strings like `/p2p/k51xyz...` in addition to the default `/p2p/Qmabc...`. There is a corresponding `--peerid-base` option to many functions that output peerIDs.
+
+## 🧮 `dag stat`
+
+Initial support has been added for the `ipfs dag stat` command. Running this command will traverse the DAG for the given root CID and report statistics. By default, progress will be shown as the DAG is traversed. Supported statistics currently include DAG size and number of blocks.
+
+```bash
+ipfs dag stat bafybeihpetclqvwb4qnmumvcn7nh4pxrtugrlpw4jgjpqicdxsv7opdm6e # the IPFS webui
+Size: 30362191, NumBlocks: 346
+```
+
+## 🔌 Plugin build changes
```

Comment:
Wondering if we should add an 🚨 or something here to help highlight it's a breaking change for some of the plugin folks. Perhaps it doesn't really matter though since you're either reading the highlights or you've already skipped them and it's too late.

Reply:
I added the alert icons around it and also added a note in the intro with an alert to check out these changes for folks using plugins.